### DISCUSSION TITLE
Rehost or internet archive page of the kernel version switcher?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1116] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does anyone know of an internet archive page for the kernel version switcher? Or at least a rehost of it?

**Comment by FunnyCorgi** [comment]
I found a page for it a long time ago, but I lost the link to it.

**Comment by AshtonDavies** [comment]
I might rehost it sometime soon. Brandon was rehosting it on his GitHub, but his account is currently suspended, which disabled all of his repositories.

**Comment by s0urce-c0de** [comment, marked as answer]
I have a privated one (that kxtzownsu asked me to take down). If you want public, look [here](https://github.com/uu5007mp/KVS).
